server/src/api/schemas.py

```diff
@@ -27,6 +27,7 @@ class AgentIn(Schema):
     state = fields.String(required=False)
     queues = fields.List(fields.String(), required=False)
     location = fields.String(required=False)
+    provision_type = fields.String(required=False)
```

Comment:
could `provision_type` be an enum? that would help with api docs too since it would show possible values

Reply:
At some point probably, it will be but that's something we just started discussing for other reasons. It will still likely need to convert to a string (string val enum, not int) though.